### PR TITLE
Updated collaborator parts of the user guide

### PR DESCRIPTION
**Pull request by mishaone**
* Updated collaborator section and added screenshots

### CODE REVIEW

File: docs/topics/getting-started-guide.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 = Before you start
 
 include::modules/introduction.adoc[leveloffset=+1]
-
+zA
```
Comment (rkratky) [inline docs/topics/getting-started-guide.adoc]: Huh?

**Comment by fabric8cd** [comment]
@mishaone snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-214-2 is available for testing at http://fabric8-online-docs-app-pr-214-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@mishaone snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-214-3 is available for testing at http://fabric8-online-docs-app-pr-214-fabric8-online-docs.badger.fabric8.io

**Comment by mishaone** [comment]
Extra keystrokes from somewhere. Sorry, I'll get rid of this tomorrow.

Regards,

Misha Husnain Ali

**Comment by rkratky** [comment]
Np, @mishaone. Got rid of it and merging.

**Comment by fabric8cd** [comment]
@mishaone snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-214-4 is available for testing at http://fabric8-online-docs-app-pr-214-fabric8-online-docs.badger.fabric8.io

**Comment by mishaone** [comment]
Great, thanks!

**Pull request merged**